primitives/staking/src/currency_to_vote.rs

```diff
@@ -0,0 +1,87 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2019-2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use sp_arithmetic::traits::{UniqueSaturatedFrom, UniqueSaturatedInto};
```

Comment:
these might already be re-exported from sp_runtime, meaning that you might not need to add `sp_arithmetic` is dependency.

Reply:
indeed they are!